is GH Actions having issues?
```
Run echo "Failed to restore Python virtual environment from cache"
Failed to restore Python virtual environment from cache
Error: Process completed with exit code 1.
```

--- a/homeassistant/components/zha/manifest.json
+++ b/homeassistant/components/zha/manifest.json
@@ -4,16 +4,16 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/zha",
   "requirements": [
-    "bellows==0.25.0",
+    "bellows==0.26.0",
     "pyserial==3.5",
     "pyserial-asyncio==0.5",
     "zha-quirks==0.0.59",
     "zigpy-cc==0.5.2",
     "zigpy-deconz==0.12.0",
-    "zigpy==0.35.2",
+    "zigpy==0.36.0",
     "zigpy-xbee==0.13.0",
     "zigpy-zigate==0.7.3",
-    "zigpy-znp==0.5.1"
+    "zigpy-znp==0.5.2"
   ],
   "codeowners": ["@dmulcahey", "@adminiuga"],
   "zeroconf": [

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -364,7 +364,7 @@ beautifulsoup4==4.9.3
 # beewi_smartclim==0.0.10
 
 # homeassistant.components.zha
-bellows==0.25.0
+bellows==0.26.0
 
 # homeassistant.components.bmw_connected_drive
 bimmer_connected==0.7.15
@@ -2455,10 +2455,10 @@ zigpy-xbee==0.13.0
 zigpy-zigate==0.7.3
 
 # homeassistant.components.zha
-zigpy-znp==0.5.1
+zigpy-znp==0.5.2
 
 # homeassistant.components.zha
-zigpy==0.35.2
+zigpy==0.36.0
 
 # homeassistant.components.zoneminder
 zm-py==0.5.2

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -220,7 +220,7 @@ azure-eventhub==5.5.0
 base36==0.1.1
 
 # homeassistant.components.zha
-bellows==0.25.0
+bellows==0.26.0
 
 # homeassistant.components.bmw_connected_drive
 bimmer_connected==0.7.15
@@ -1355,10 +1355,10 @@ zigpy-xbee==0.13.0
 zigpy-zigate==0.7.3
 
 # homeassistant.components.zha
-zigpy-znp==0.5.1
+zigpy-znp==0.5.2
 
 # homeassistant.components.zha
-zigpy==0.35.2
+zigpy==0.36.0
 
 # homeassistant.components.zwave_js
 zwave-js-server-python==0.28.0

--- a/tests/components/zha/test_api.py
+++ b/tests/components/zha/test_api.py
@@ -136,7 +136,7 @@ async def test_device_cluster_attributes(zha_client):
     msg = await zha_client.receive_json()
 
     attributes = msg["result"]
-    assert len(attributes) == 4
+    assert len(attributes) == 5
 
     for attribute in attributes:
         assert attribute[ID] is not None

